src/constants.ts

```diff
@@ -62,6 +59,120 @@ export const URL_SCHEME_OPEN = 'openUrl';
 export const SHARED_PREF_USER_ID = 'userId';
 export const SHARED_PREF_EMAIL = 'email';
 
+export type RouteConfig = {
+  route: string;
+  body: boolean; // true for POST/PUT requests
+  current: boolean; // true if email or userId in request needs to be prepended with `current`
+  nestedUser: boolean; // true if route expects email or userId field to be nested in user object
+};
```

Comment:
Rec to use jsdoc format so that hovering over properties can show these comments elsewhere
```suggestion
export type RouteConfig = {
  route: string;
  /** true for POST/PUT requests */
  body: boolean;
  /** true if email or userId in request needs to be prepended with `current` */
  current: boolean;
  /** true if route expects email or userId field to be nested in user object */
  nestedUser: boolean;
};
```